packages/flutter_tools/test/general.shard/asset_bundle_variant_test.dart

```diff
@@ -194,46 +223,18 @@ flutter:
       );
 
       final Map<String, List<String>> manifest = await extractAssetManifestFromBundle(bundle);
-      final List<String> variantsForImage = manifest[image]!;
-
-      expect(variantsForImage, contains(image2xVariant));
-      expect(variantsForImage, isNot(contains(imageNonVariant)));
-    });
 
-    testWithoutContext('Asset directories are recursively searched for assets', () async {
-      const String topLevelImage = 'assets/image.jpg';
-      const String secondLevelImage = 'assets/folder/secondLevel.jpg';
-      const String secondLevel2xVariant = 'assets/folder/2x/secondLevel.jpg';
-
-      final List<String> assets = <String>[
-        topLevelImage,
-        secondLevelImage,
-        secondLevel2xVariant
+      final List<String> expectedFooVariants = <String>[
+        withPosixPathSeparators(foo),
```

Comment:
Can we hard code the expectations, rather than processing them? It's difficult for me reading this code exactly what we're expecting the manifest to have.

Reply:
also on lines 232,233